### PR TITLE
feat: dockerfile

### PR DESCRIPTION
**Pull request by batrlatom**

### CODE REVIEW

File: Docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM nvidia/cuda:11.6.2-cudnn8-devel-ubuntu20.04
+RUN apt-get update -y
+RUN apt-get upgrade -y
+RUN apt-get install python3 -y
+RUN apt-get install python3-pip -y
+RUN apt-get install git -y
+RUN pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html
+RUN pip install -q git+https://github.com/borisdayma/dalle-mini.git
+RUN pip install -q git+https://github.com/patil-suraj/vqgan-jax.git
+WORKDIR /workspace
```
Comment (Saibamen) [inline Docker/Dockerfile:10]:
```suggestion
FROM nvidia/cuda:11.6.2-cudnn8-devel-ubuntu20.04

RUN apt-get update && apt-get install -y \
  git \
  python3 \
  python3-pip \
  && rm -rf /var/lib/apt/lists/*

RUN pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html \
  && pip install -q \
  git+https://github.com/borisdayma/dalle-mini.git \
  git+https://github.com/patil-suraj/vqgan-jax.git

WORKDIR /workspace

```

**Review by Saibamen — CHANGES_REQUESTED** [review]
```suggestion
FROM nvidia/cuda:11.6.2-cudnn8-devel-ubuntu20.04

RUN apt-get update && apt-get install -y \
  git \
  python3 \
  python3-pip \
  && rm -rf /var/lib/apt/lists/*

RUN pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html \
  && pip install -q \
  git+https://github.com/borisdayma/dalle-mini.git \
  git+https://github.com/patil-suraj/vqgan-jax.git

WORKDIR /workspace

```

Use dockerfile best practices: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

**Comment by borisdayma** [comment]
Many thanks!
Is it ready to be merged?

**Comment by batrlatom** [comment]
The dockerfile was intented to be just a environment for development and inference. If you want to have for example gradio or streamlit app to be run after the start, I can make it so.

**Pull request merged**